docs/_articles/en/einstein/einstein-inline.md

```diff
@@ -18,6 +18,16 @@ To enable or disable the Inline Autocomplete feature from Settings:
 1. Select **File** > **Preferences** > **Settings** (Windows or Linux) or **Code** > **Preferences** > **Settings** (macOS).
 2. Under **Einstein for Developers**, select **Enable Einstein Auto Completions** then select the language for which to enable the feature.
 
+## Select Inline Completion Length
+
+To select the length of generated inline autocompletions:
+
+1. Select **File** > **Preferences** > **Settings** (Windows or Linux) or **Code** > **Preferences** > **Settings** (macOS).
```

Comment:
Is it 
```suggestion
1. Select **File** > **Preferences** > **Settings** (Windows or Linux) or **Code** > **Settings** > **Settings** (macOS).
```
for Mac?
<img width="535" alt="2024-05-16_10-50-14" src="https://github.com/forcedotcom/salesforcedx-vscode/assets/46458081/45fddd10-b93b-4a85-b937-2c8b4de9ce88">